apps/hash-graph/lib/graph/src/api/rest.rs

```diff
@@ -489,22 +477,34 @@ impl Modify for TimeSchemaAddon {
                     .into(),
             );
             components.schemas.insert(
-                UnboundedOrExcludedTimeIntervalBound::<()>::schema()
-                    .0
-                    .to_owned(),
-                UnboundedOrExcludedTimeIntervalBound::<()>::schema().1,
+                TemporalBound::<()>::schema().0.to_owned(),
+                TemporalBound::<()>::schema().1,
             );
             components.schemas.insert(
-                IncludedTimeIntervalBound::<()>::schema().0.to_owned(),
-                IncludedTimeIntervalBound::<()>::schema().1,
+                LimitedTemporalBound::<()>::schema().0.to_owned(),
+                LimitedTemporalBound::<()>::schema().1,
             );
             components.schemas.insert(
-                TimeIntervalBound::<()>::schema().0.to_owned(),
-                TimeIntervalBound::<()>::schema().1,
+                OpenTemporalBound::<()>::schema().0.to_owned(),
+                OpenTemporalBound::<()>::schema().1,
             );
             components.schemas.insert(
-                LimitedTimeIntervalBound::<()>::schema().0.to_owned(),
-                LimitedTimeIntervalBound::<()>::schema().1,
+                ClosedTemporalBound::<()>::schema().0.to_owned(),
+                ClosedTemporalBound::<()>::schema().1,
+            );
+            components.schemas.insert(
+                "LeftClosedTemporalInterval".to_owned(),
+                LeftClosedTemporalInterval::<()>::schema().1,
+            );
+            components.schemas.insert(
+                "RightBoundedTemporalInterval".to_owned(),
```

Comment:
Any reason for us not to use the `.0` element of the tuple here for the name?

Reply:
This is only a type alias, which would refer to `"Interval"`, but we want it to be `"RightBoundedTemporalInterval".